spec/lib/authorities/linked_data/search_config_spec.rb

```diff
@@ -12,7 +13,7 @@
         url: {
           :@context => 'http://www.w3.org/ns/hydra/context.jsonld',
           :@type => 'IriTemplate',
-          template: 'http://localhost/test_default/search?{?subauth}&{?query}&{?param1}&{?param2}',
+          template: 'http://localhost/test_default/search?{?subauth}&{?query}&{?param1}&{?param2}&{?lang}',
```

Comment:
The updates to the expected `full_config`  here corresponds to the changes to the `lod_full_config.json` fixture.